setup.py

```diff
@@ -19,6 +19,7 @@
 AUTHOR = "Henrik Blidh"
 
 REQUIRED = [
+    "async_timeout>=4.0.1",
```

Comment:
There is also a pipenv file with yet another copy of the requirements.

Reply:
I added the changes to `Pipfile`, but I didn't find any other place.  Is there another file?